test/shopify_api/rest/tag_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule ShopifyAPI.REST.TagTest do
+  use ExUnit.Case
+  use ExUnitProperties
+
+  alias ShopifyAPI.REST.Tag
+
+  describe "Tags.encode and split" do
```

Comment:
This `describe` block needs to be renamed, and can probably be folded in with the other one.

Reply:
oh, yea! good catch!